source/frontend/src/components/common/mapFSM/machineDefinition/mapMachine.ts

```diff
@@ -372,20 +343,50 @@ const sideBarStates = {
 };
 
 const advancedFilterSideBarStates = {
-  initial: 'fullScreen',
+  initial: 'close',
   states: {
-    fullScreen: {
+    close: {
+      on: {
+        TOGGLE_FILTER: {
+          target: 'filtering',
+        },
+        TOGGLE_LAYERS: {
+          target: 'layerControl',
+        },
+      },
+    },
+    layerControl: {
       on: {
-        OPEN_ADVANCED_FILTER_SIDEBAR: {
-          target: 'sidebarOpen',
+        TOGGLE_FILTER: {
+          target: 'filtering',
+        },
+        TOGGLE_LAYERS: {
+          target: 'close',
+        },
+        SET_MAP_LAYERS: {
+          actions: assign({ activeLayers: (_, event: any) => event.activeLayers }),
         },
       },
     },
-    sidebarOpen: {
-      entry: assign({ mapFilter: () => defaultPropertyFilter }),
+    filtering: {
+      entry: [send({ type: 'REFRESH_PROPERTIES', searchCriteria: defaultPropertyFilter })],
+      exit: [send({ type: 'REFRESH_PROPERTIES' })],
       on: {
-        CLOSE_ADVANCED_FILTER_SIDEBAR: {
-          target: 'fullScreen',
+        TOGGLE_FILTER: {
+          target: 'close',
+          actions: [assign({ showDisposed: () => false }), assign({ showRetired: () => false })],
```

Comment:
I'm on the fence if it makes sense for the state machine to enforce defaults like this, or if it should be up to the component itself.

Reply:
Also, agree. But this was like that before, rather keep the logic as is for existing filter parameters